tier4_perception_msgs/msg/traffic_light/TrafficLightRoi.msg

```diff
@@ -1,2 +1,6 @@
+uint8 CAR_TL=0
+uint8 PEDESTRIAN_TL=1
```

Comment:
```suggestion
uint8 CAR_TRAFFIC_LIGHT=0
uint8 PEDESTRIAN__TRAFFIC_LIGHT=1
```
not to use shorten word

Reply:
fixed in https://github.com/tier4/tier4_autoware_msgs/commit/1c10e3b4af2f50c95863b53c8bb889daf9db95c6

Reply:
[1c10e3b](https://github.com/tier4/tier4_autoware_msgs/pull/110/commits/1c10e3b4af2f50c95863b53c8bb889daf9db95c6)